src/styles/catalog.scss

```
@@ -0,0 +1,123 @@
+.catalog {
+  display: flex;
+  margin: 0 auto;
+  flex-wrap: wrap;
+  width: $catalog-width;
+  padding: 51px 0 0;
+}
+
+.card {
+  background: #fff;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  font-size: 10px;
```

I think it was the cllass where we should begin using right property order
- [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) Number 12

I've seen other moments based on this rule further in code but I will only leave this comment for you to fix all issues.
```suggestion
  display: flex;
  flex-direction: column;
  width: 200px;
  padding: 32px 16px 16px;
  
  font-size: 10px;
  
  background: #fff;
  border: 1px solid #f3f3f3;
  border-radius: 5px;
```